src/dune_rules/lib_file_deps.ml

```diff
@@ -5,25 +5,32 @@ module Group = struct
     | Cmi
     | Cmx
 
+  type melange =
+    | Cmi
+    | Cmj
+    | Js
```

Comment:
Let's keep the melange type definitions in `Melange` or `Melange_rules` if possible.

Reply:
Moved to `Melange`, as `Melange_rules` would lead to dep cycle:
```
-> Compilation_context
-> Lib_file_deps
-> Melange_rules
-> Compilation_context
```

https://github.com/ocaml/dune/pull/6349/commits/bfaf62e5444ab5b2e380dcb3413b3f7aea492244